src/ImageRunModal.jsx

```diff
@@ -129,6 +130,19 @@ const Volume = ({ id, item, onChange, idx, removeitem, additem }) =>
                         {_("ReadWrite")}
                     </Select.SelectEntry>
                 </Select.Select>
+                <Select.Select className='form-control'
+                               initial={item.mode}
+                               onChange={value => onChange(idx, 'selinux', value)}>
+                    <Select.SelectEntry data='' key=''>
+                        {_("None SELinux label")}
```

Comment:
"No SELinux label". "None" is indefinite, and ungrammatical in this context.

Reply:
Fixed.